Assets/Scripts/Gameplay/Mecha/MechaController.cs

```diff
@@ -54,6 +54,11 @@ public enum View
         [SerializeField] private float recoilDuration = 0.1f;
         [SerializeField] private float recoilCounterForce = 1f;
 
+        [Header("Dash")]
+        [SerializeField] private float dashSpeed = 2000f;   // Dash force applied during dash
+        [SerializeField] private float dashDuration = 0.5f;  // How long dash lasts
+        [SerializeField] private float dashCooldown = 1f;    // CD time before next dash
+
```

Comment:
These values could be move to [JuggernautParameters.cs](https://github.com/Taliayaya/Project-86/blob/main/Assets/Scripts/ScriptableObjects/GameParameters/JuggernautParameters.cs)